<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainDialog</class>
    <widget class="QDialog" name="MainDialog">
        <property name="windowModality">
            <enum>Qt::NonModal</enum>
        </property>
        <property name="windowFlags">
            <set>Qt::Window</set>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>700</width>
                <height>700</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>700</width>
                <height>700</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Images Viewer</string>
        </property>
        <layout class="QVBoxLayout" name="mainVerticalLayout">
            <property name="margin">
                <number>0</number>
            </property>
            <property name="spacing">
                <number>0</number>
            </property>
            <item>
                <layout class="QHBoxLayout" name="topBarLayout">
                    <property name="margin">
                        <number>0</number>
                    </property>
                    <item>
                        <widget class="QToolBar" name="topToolBar">
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QScrollArea" name="mainScrollArea">
                    <property name="widgetResizable">
                        <bool>true</bool>
                    </property>
                    <widget class="QWidget" name="scrollAreaContents">
                        <layout class="QGridLayout" name="gridLayout">
                            <property name="spacing">
                                <number>25</number>
                            </property>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="bottomBarLayout">
                    <property name="margin">
                        <number>5</number>
                    </property>
                    <property name="spacing">
                        <number>5</number>
                    </property>
                    <item>
                        <widget class="QComboBox" name="featuresFilterComboBox">
                        </widget>
                    </item>
                    <item>
                        <spacer name="bottomBarSpacerLeft">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>1</width>
                                    <height>1</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="paginationButtonsLayout">
                            <item>
                                <widget class="QPushButton" name="previousPageButton">
                                    <property name="enabled">
                                        <bool>false</bool>
                                    </property>
                                    <property name="text">
                                        <string> Previous</string>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>150</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="icon">
                                        <iconset resource="../../images/images.qrc">
                                            <normaloff>:/images/themes/default/mActionArrowLeft.svg</normaloff>
                                            :/images/themes/default/mActionArrowKLeft.svg
                                        </iconset>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QPushButton" name="nextPageButton">
                                    <property name="enabled">
                                        <bool>false</bool>
                                    </property>
                                    <property name="text">
                                        <string>Next </string>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>150</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="layoutDirection">
                                        <enum>Qt::RightToLeft</enum>
                                    </property>
                                    <property name="icon">
                                        <iconset resource="../../images/images.qrc">
                                            <normaloff>:/images/themes/default/mActionArrowRight.svg</normaloff>
                                            :/images/themes/default/mActionArrowRight.svg
                                        </iconset>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <spacer name="bottomBarSpacerRight">
                            <property name="orientation">
                                <enum>Qt::Horizontal</enum>
                            </property>
                            <property name="sizeType">
                                <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="sizeHint" stdset="0">
                                <size>
                                    <width>1</width>
                                    <height>1</height>
                                </size>
                            </property>
                        </spacer>
                    </item>
                    <item>
                        <widget class="QComboBox" name="relationComboBox">
                            <property name="toolTip">
                                <string>Select relationship</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QgsFieldComboBox" name="fieldComboBox">
                            <property name="toolTip">
                                <string>Select field containing image data or url</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QgsFieldComboBox</class>
            <extends>QComboBox</extends>
            <header>qgsfieldcombobox.h</header>
        </customwidget>
    </customwidgets>
    <resources />
    <connections />
</ui>